centreon/www/front_src/src/ResourceAccessManagement/AddEditResourceAccessRule/AddEditResourceAccessRuleModal.tsx

```diff
@@ -0,0 +1,53 @@
+import { ReactElement, useMemo } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import { Typography } from '@mui/material';
+
+import { Modal } from '@centreon/ui/components';
+
+import {
+  labelCreateResourceAccessRule,
+  labelEditResourceAccessRule
+} from '../translatedLabels';
+
+import useResourceAccessRuleModal from './useResourceAccessRuleModal';
+import useModalStyles from './Modal.styles';
+import Form from './Form/Form';
```

Comment:
Maybe we can change the name of file from Form to index.tsx and we can import component like this: 
``import Form from './Form``. This gives more lisibility

Reply:
Done